frontend/pages/g/_groupSlug/r/_slug/index.vue

```diff
@@ -32,28 +32,28 @@ export default defineComponent({
       const api = usePublicExploreApi(groupSlug.value);
       recipe = useAsync(async () => {
         const { data, error } = await api.explore.recipes.getOne(slug);
-
         if (error) {
           console.error("error loading recipe -> ", error);
           router.push(`/g/${groupSlug.value}`);
         }
 
+        setMeta(data);
```

Comment:
the metadata will not be set when the recipe belongs to the group of the user

Reply:
Ah you're right, I must've been logged out while testing this. Thanks!
The reason it's in the branch like this is because the metadata wasn't getting set after the async call (so the title was never set). Just fixed it for the logged-in branch using a watcher

Reply:
why have the watcher in the if _isOwnGroup_ case and not just put it after the if-else statement. Could delete the `setMeta(data)` in the else block

Reply:
Huh, for some reason I thought that didn't work because the owner block replaces the variable (rather than updating the ref data), but I tested it and it does work, so never mind